ui/admin/app/controllers/scopes/scope/workers/worker/create-tags.js

```diff
@@ -41,8 +41,6 @@ export default class ScopesScopeWorkersWorkerCreateTagsController extends Contro
    */
   @action
   cancel() {
-    // Clear the apiTags array when canceling
-    this.apiTags = new TrackedArray([]);
```

Comment:
Isn't this behavior different? Don't we normally just rollback the changes and let the user cancel instead of a popup appearing when trying to cancel?

Reply:
Yep that is right. Changing back.